engine/explain.go

```diff
@@ -61,13 +61,25 @@ func (a *AnalyzeOutputNode) aggregateSamples() {
 		for _, child := range a.Children {
 			childPeak := child.PeakSamples()
 			a.peakSamples = max(a.peakSamples, childPeak)
+
+			if a.OperatorTelemetry.SubQuery() {
+				// Skip aggregating from subquery to avoid double counting samples from children.
+				continue
+			}
+
+			if a.OperatorTelemetry.Invariant() {
+				// Invariant operators should have only one samples but they should be counted across all the steps
```

Comment:
What does "should have only one samples" mean here?
```suggestion
				// Invariant operators should have only one samples but they should be counted across all the steps
```